src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -107,18 +103,21 @@ class ReadOnlyMessage extends Component {
   }
 }
 
-class NotebookPreview extends Component {
-  render() {
-    const { namespace, name } = this.props
-    return h(Fragment, [
-      linkButton({
-        style: { position: 'absolute', top: 20, left: 'calc(50% + 570px)' },
-        onClick: () => Nav.goToPath('workspace-notebooks', { namespace, name })
-      }, [icon('times-circle', { size: 30 })]),
-      h(NotebookPreviewFrame, this.props)
-    ])
-  }
-}
+const NotebookPreview = pure(props => {
+  const { namespace, name, notebookName, workspace } = props
+  return h(Fragment, [
+    h(ReadOnlyMessage, { notebookName, workspace }),
+    h(NotebookPreviewFrame, {
+      ...props,
+      loadedChildren: div({ style: { position: 'relative' } }, [
```

Comment:
So it doesn't appear before the frame, which looked weird

Reply:
What is `loadedChildren` used for?